src/en/controllers-creating.md

```diff
@@ -1,50 +1,75 @@
 Title: Creating a Juju Controller
-TODO: Add better examples
+TODO:  Improve examples
+       Hardcoded: Ubuntu code names
 
 
 # Creating a controller
 
 Use the `juju bootstrap` command to create a controller (and model) for a given
 cloud:
 
-`juju bootstrap [options] <cloud name> <controller name>`
+`juju bootstrap [options] <cloud name> [<controller name>]`
 
 See `juju help bootstrap` for details on this command or see the
-[command reference page](./commands.html#juju-bootstrap).
+[Juju command reference][commands] page.
 
 The `<controller name>` is optional. If one is not supplied, then a name is
 assigned based on the cloud and region.
 
+To learn about configuration options available at bootstrap time, see
+[Configuring controllers][controlconfig] and [Configuring models][modelconfig].
+
+## Model constraints
```

Comment:
I wouldn't put this new material  before the examples, or relegate the other things to 'Other' examples. This sort of suggests that model constraints and controller constraints are the most important options (which they aren't) and forces the reader to go past this to get to the examples.

I would rather have these as actual examples. E.g.  'Create a controller with at least 4GB memory'

Reply:
I moved things around a bit. Lemme know.